packages/palette/src/elements/Link/Link.tsx

```diff
@@ -29,17 +31,22 @@ const backwardsCompatCompute = (state: string, props: LinkProps) => {
 /**
  * Basic <a> tag styled with additional LinkProps
  *
- * @deprecated Do not use this component! 
- * Tip: If working on Force, please use RouterLink.  
+ * Tip: If working on Force, please use RouterLink.
  */
 export const Link = styled.a<LinkProps>`
-  color: ${color("black100")};
+  color: inherit;
```

Comment:
Correct me if I’m wrong but if we are using as={Link}  we can’t also pass it a prop to specify the color. So I changed this to `inherit` as the design specifies that the link color should be either black100 or black60 depending on the text color of the component, but let me know if you have a better solution to this.

Reply:
`currentColor` would be better here.